src/plugins/intel_gpu/src/graph/impls/cpu/non_max_suppression.cpp

```diff
@@ -375,6 +375,15 @@ void run(non_max_suppression_inst& instance) {
         return;
     }
 
+    if (instance.outputs_memory_count() == 3)
```

Comment:
Could you please add some comments of difference b/w the above codes? 
e.g., 
// Legacy method using mutable inputs for multiple outputs
~~~
// New method using multiple outputs' support 
~~~~

Reply:
done